configuration.go

```diff
@@ -46,10 +46,12 @@ const (
 	BundlePolicyMaxCompat
 )
 
-var BundlePolicyString = []string{"Balanced", "MaxBundle", "MaxCompat"}
-
 func (p BundlePolicy) String() string {
-	return EnumToStringSafe(int(p), BundlePolicyString)
+	return EnumToStringSafe(int(p), []string{
+		"Balanced",
```

Comment:
I agree it would be a micro-optimization, also considering it's unlikely that String will be called in performance-critical code.

But yes, lower-casing the vars would also work.